pkgs/by-name/wh/whistle/package.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, buildNpmPackage, fetchFromGitHub }:
+
+buildNpmPackage rec {
+  pname = "whistle";
+  version = "2.9.58";
+
+  src = fetchFromGitHub {
+    owner = "avwo";
+    repo = "whistle";
+    rev = "7dd832d925422ae30efe3d26e78eb8dc73a34a7b"; # release not tagged
```

Comment:
There is a `v2.9.58` tag: https://github.com/avwo/whistle/tree/v2.9.58 ?

Reply:
Oops. Fixed, Thanks!